sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -884,6 +902,31 @@ SIREPO.app.directive('msieFontDisabledDetector', function(errorService, $interva
     };
 });
 
+SIREPO.app.directive('safePath', function() {
+    return {
+        restrict: 'A',
+        require: 'ngModel',
+        link: function(scope, element, attrs, ngModel, $sce) {
+
+            scope.showWarning = false;
+            scope.warningText = scope.info[0] + ' must not include any of the following: ' +
```

Comment:
I computed this once. I don't think it wants to be computed every time so rather:

```javascript
scope.warningText = '';
```